infrastructure-playbooks/switch-from-non-containerized-to-containerized-ceph-daemons.yml

```diff
@@ -91,13 +91,8 @@
       when: ceph_docker_image is search("rhceph")
 
     - name: set proper ownership on ceph directories
-      file:
-        path: "{{ item }}"
-        owner: "{{ ceph_uid }}"
-        recurse: yes
-      with_items:
-        - /var/lib/ceph
-        - /etc/ceph
+      command: "chown -R {{ ceph_uid }} /var/lib/ceph /etc/ceph"
```

Comment:
Can you add a bit of rationale in the code so that we know why we are doing this even though it's not ideal? Also, that'll prevent people from reverting this later. Thanks.

Reply:
Added a note above each set proper ownership task explaining change from file module to raw command